src/sisl/io/xyz.py

```diff
@@ -100,10 +100,10 @@ def _r_geometry_skip(self, *args, **kwargs):
         return na
 
     @sile_fh_open()
-    @sile_read_multiple(skip_call=_r_geometry_skip, postprocess=GeometryCollection)
+    @sile_read_multiple(stop=1, skip_call=_r_geometry_skip, postprocess=GeometryCollection)
```

Comment:
It would somehow be cleaner if the default was always to return everything, unless the user provides some selection with `start/stop/step`.